<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calorie Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="signin">
       <attribute name="title">
        <string>Sign in</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>250</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pbSignin">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>130</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Sign in</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pbGotoSignup">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>170</y>
             <width>141</width>
             <height>23</height>
            </rect>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>No account - signup</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lbLogin_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbPassword_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lePassword">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>90</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLineEdit" name="leLogin">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>50</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>10</y>
             <width>50</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#0000ff;&quot;&gt;Sign in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="signup">
       <attribute name="title">
        <string>Sign up</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>290</width>
            <height>290</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>290</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLineEdit" name="lePassword1">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>160</y>
             <width>120</width>
             <height>20</height>
            </rect>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="lbLoginReg">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>40</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="deBirthdayReg">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>100</y>
             <width>120</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="leNameReg">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>70</y>
             <width>120</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lbPassword2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>190</y>
             <width>91</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Confirm password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lePassword2">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>190</y>
             <width>120</width>
             <height>20</height>
            </rect>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="lbName">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>70</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbPassword">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>160</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="leLoginReg">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>40</y>
             <width>120</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="lbBirthday">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>100</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>Birthday</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pbSignup">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>230</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Sign up</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>10</y>
             <width>50</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#0000ff;&quot;&gt;Sign up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbGender">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>130</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QComboBox" name="cbGenderReg">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>130</y>
             <width>120</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>female</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>male</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSignIn"/>
    <addaction name="actionSignUp"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionFood"/>
   </widget>
   <widget class="QMenu" name="menuQueries">
    <property name="title">
     <string>Queries</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuData"/>
   <addaction name="menuQueries"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionFood">
   <property name="text">
    <string>Food</string>
   </property>
  </action>
  <action name="actionSignIn">
   <property name="text">
    <string>Sign in</string>
   </property>
  </action>
  <action name="actionSignUp">
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pbSignin</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pbGotoSignup</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leLogin</tabstop>
  <tabstop>lePassword1</tabstop>
  <tabstop>deBirthdayReg</tabstop>
  <tabstop>leNameReg</tabstop>
  <tabstop>lePassword2</tabstop>
  <tabstop>leLoginReg</tabstop>
  <tabstop>pbSignup</tabstop>
 </tabstops>
 <resources>
  <include location="foodcl.qrc"/>
 </resources>
 <connections/>
</ui>
